docs/monitoring/processes.md

```diff
@@ -1,12 +1,87 @@
 title: Sematext Process Monitoring
 description: Sematext Inventory Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
 
-This should explain the new Inventory Monitoring feature, and have the following sections.
+Process Monitoring gives you visibility into processes that use the most CPU or memory, regardless of whether they are running inside containers, in the cloud, your own VMs or bare-metal servers.
 
-- overview
-- how it works
-- what it does
-- what it solves 
-- how it helps with observability/troubleshooting/debugging/etc…
+The process information is available in the Monitoring *Infrastructure* section of your Sematext Cloud account - your central place for hosts, virtual machines, containers, and process information. 
+
+### Process View
+
+The process view provides you similar information as the Linux `top` command, but for your entire infrastructure: 
```

Comment:
The process view provides you similar information as the Linux `top` command, but for your entire infrastructure: 
->
The process view provides similar information as the Linux `top` command. It shows: